lib/rails_cloudflare_turnstile/view_helpers.rb

```diff
@@ -25,6 +30,15 @@ def turnstile_div(action)
       end
     end
 
+    def mock_turnstile_div(action)
+      content_tag(:div, class: "cf-turnstile", style: "width: 300px; height: 65px: border: 1px solid gray") do
```

Comment:
we should throw a `class` on this baddy, rather than `style`

Reply:
I don't actually know how to inject CSS from a rails plugin into the main app. I assume there's some way to provide a SASS file that goes into the build flow?

Reply:
Ah, yea, this is an oddball here. Hmmmmmm... we could use a `stylesheet_link_tag`, but this is feeling overkill-y.

Reply:
yeah I think for now maybe we just leave the style tag on it?